Dockerfile

```diff
@@ -18,9 +18,11 @@ COPY --from=base /isaac-api/target/isaac-api.war /var/lib/jetty/webapps/isaac-ap
 RUN chmod 755 /var/lib/jetty/webapps/*
 RUN chown jetty /var/lib/jetty/webapps/*
 
-COPY resources/jetty.xml /usr/local/jetty/etc/
 COPY resources/start.ini /var/lib/jetty/
 
 # prepare things so that jetty runs in the docker entrypoint
 USER jetty
 WORKDIR $JETTY_BASE
+
+# enable jetty modules for ee9
+RUN java -jar "$JETTY_HOME/start.jar" --add-modules=ee9-webapp,ee9-deploy,ee9-jsp,ee9-jstl,ee9-websocket-jetty,ee9-websocket-jakarta
```

Comment:
Could you explain how this works, please? I don't understand it. It looks like it's starting Jetty with a series of modules, that's fine. But what I don't understand is how come this new line is necessary? We didn't have to manually execute the server before (by running `start.jar`), so I don't get why we would need to do it now, if that makes sense. Also, there could be another way of achieving the same thing. For instance, if all you need is to ensure that the modules are loaded, can't you just update the `maven-compiler-plugin` in `pom.xml` and add the arguments there, inside `<configuration>`? Like this:

```xml
<compilerArgs>
    <arg>--add-modules</arg>
    <arg>ee9-webapp,ee9-deploy,etc.</arg>
</compilerArgs>
```

Similarly, I don't understand why we need this line in this file, but not in the `-api` and `-etl` Dockerfiles.

Reply:
That might be a neater way of doing it, I'll give it a try, thanks

Reply:
the compilerArguments approach didn't work (because these are params to start.jar rather than mvn) but it did prompt me to find a better way.

I've now added all the required ini files to start the modules.